app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def calculate_team_total_rating(team):
+    return sum([i.get_rating() for i in team])
+
+
+def elves_concert(elves):
+    for i in elves:
```

Comment:
You should use more infomative names, than "i". It's still solid code but can be hard for understanding to the others